nixos/modules/hardware/video/nvidia.nix

```
@@ -79,6 +79,14 @@ in
       '';
     };
 
+    hardware.nvidia.optimus_prime.external_gpu = lib.mkOption {
```

```suggestion
    hardware.nvidia.optimus_prime.allowExternalGpu = lib.mkOption {
```

Makes sense and is camel cased.